libs/galley-types/src/Galley/Types/Teams.hs

```diff
@@ -150,7 +151,8 @@ data FeatureFlags = FeatureFlags
     _flagsTeamFeatureValidateSAMLEmailsStatus :: !(Defaults (ImplicitLockStatus ValidateSAMLEmailsConfig)),
     _flagTeamFeatureSndFactorPasswordChallengeStatus :: !(Defaults (WithStatus SndFactorPasswordChallengeConfig)),
     _flagTeamFeatureSearchVisibilityInbound :: !(Defaults (ImplicitLockStatus SearchVisibilityInboundConfig)),
-    _flagMLS :: !(Defaults (ImplicitLockStatus MLSConfig))
+    _flagMLS :: !(Defaults (ImplicitLockStatus MLSConfig)),
+    _flagTeamFeatureExposeInvitationURLsTeamAllowlist :: !ExposeInvitationURLsTeamAllowlistConfig
```

Comment:
This shouldn't be in `FeatureFlags`, as it doesn't have the "status", "lockStatus", "otherStuff" structure and re. classes etc.

Move to Galley.Options?  Probably in Settings?